src/bootlint.js

```diff
@@ -938,6 +938,16 @@ var LocationIndex = _location.LocationIndex;
             reporter('`.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`', formControlsOnWrongTypes);
         }
     });
+    addLinter("E043", function lintNavbarButtons($, reporter) {
+        $('.navbar-nav').find('a').each(function () {
```

Comment:
Refactored. I see what you mean about letting the browser do the heavy lifting. I hadn't thought of that. Thanks for the critique on that :smiley: